### PR TITLE
Audio: crossover: not clean out configs in reset

### PR DESCRIPTION
**Pull request by eddy1021**
configs from userspace shouldn't be cleaned out on reset.

### CODE REVIEW

File: src/audio/crossover/crossover.c [hunk 1]
```diff
@@ -837,8 +837,6 @@ static int crossover_reset(struct comp_dev *dev)
 
 	cd->crossover_process = NULL;
 	cd->crossover_split = NULL;
-	cd->config = NULL;
```
Comment (cujomalainey) [inline src/audio/crossover/crossover.c:840]: You appear to be deleting the same line, I think you might need to rebase

**Review by cujomalainey — CHANGES_REQUESTED** [review]

File: src/audio/crossover/crossover.c (hunk 1 above)
Comment (eddy1021) [reply src/audio/crossover/crossover.c:840]: The other one is `config_new` which should be resolved in later PR(switch to use new model_handler).

Comment (cujomalainey) [reply src/audio/crossover/crossover.c:840]: im open to merging the PRs but this on its own wont work

**Comment by eddy1021** [comment]
drop this PR since it's covered by pulls #5729

**Pull request closed without merging**